client/components/Reviews/Reviews.jsx

```diff
@@ -1,30 +1,58 @@
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
+import styled from 'styled-components';
 import ReviewsList from './ReviewsList';
 
-const Reviews = ({ reviews }) => {
-  const [num, setNum] = useState(2);
+const Reviews = ({ reviews, sort, setSort }) => {
+  const [num, setNum] = useState(reviews.length < 2 ? reviews.length : 2);
 
-  const updateNum = () => {
+  const handleSelect = (e) => {
+    setNum(reviews.length < 2 ? reviews.length : 2);
+    setSort(Number(e.target.value));
+  };
+
+  const loadMoreReviews = () => {
     if (num + 2 < reviews.length) {
       setNum(num + 2);
     } else {
       setNum(reviews.length);
     }
   };
 
+  const Button = styled.button`
+    background-color: inherit;
+    padding: 10px;
+    border: 1px solid black;
+    margin: 5px;
+    cursor: pointer;
+    &:active {
+      transform: translateY(3px);
+    }
+  `;
+
+  const SelectSort = styled.select`
+    background-color: inherit;
+    border: none;
+    text-decoration: underline;
+  `;
+
   return (
     <div data-testid="reviews">
       <div>
         {`${reviews.length} reviews, sorted by `}
-        <button type="button">relevance</button>
+        {/* eslint-disable-next-line styled-components-a11y/no-onchange */}
```

Comment:
Is this next-line comment something we should add to the .eslintrc, or is it being temporarily used here? No action needs to be taken for this, just something to consider.

Reply:
It is temporarily being used. I am going to try to find a way to implement the select options without needing onchange, but we that will be something for later.